scripts/install_torchbenchmark.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+if [ "$1" = "-h" -o "$1" = "--help" ]; then
+  echo "Usage: prog BRANCH_NAME REPO"
+  exit 1
+fi
+
+if [ "$#" -ne 2 ]; then
+    echo echo "Usage: prog BRANCH_NAME REPO"
+    exit 1
+fi
+
+. ${CONDA_EXE%bin/conda}/etc/profile.d/conda.sh
+ORIG_BRANCH_NAME=$1
+REPO=$2
+BRANCH_NAME=$(python -c 'import sys; print (sys.argv[1].replace("/","_"))' $ORIG_BRANCH_NAME)
+echo "Your branch will be built into $BRANCH_NAME and torchbenchmark will be installed in ${BRANCH_NAME}/benchmark"
+conda create -y -n $BRANCH_NAME python=3.7 2>&1 > /dev/null
+test $? -eq 0 || { echo "failed to create env ${BRANCH_NAME}!"; exit; }
+conda activate $BRANCH_NAME
+echo "Installing PyTorch dependencies"
+conda install -y numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses 2>&1 > /dev/null
+conda install -y -c pytorch magma-cuda100 2>&1 > /dev/null
+echo "Checking out $"
+git clone $REPO $BRANCH_NAME
+cd $BRANCH_NAME
+git checkout $ORIG_BRANCH_NAME
+git submodule sync 2>&1 > /dev/null
+git submodule update --init --recursive 2>&1 > /dev/null
+echo "Builing PyTorch"
+export USE_CUDA=1
+export BUILD_CAFFE2_OPS=0
+export USE_XNNPACK=0
+export USE_MKLDNN=1
```

Comment:
I recommend a few more build options:
Set cmake prefix to properly pick up MKL from conda:
```
CMAKE_PREFIX_PATH=$CONDA_PREFIX
```

LLVM if you want CPU fusion (this could be optional?)
```
USE_LLVM=/usr/lib/llvm-9/
```
I don't know if these are needed or if they're automatic:
```
USE_MKL=1
USE_CUDNN=1
```

Reply:
@bertmaher I wonder if I should just remove any opinionated defaults? The chances are these are already set in one's environment?

Reply:
I kinda prefer to have them in a script like this.  I'd rather have a script do the right thing than depend on random people knowing the right environmental magic.

Reply:
added them back and added optional `USE_LLVM`